students/stachowska_agata/lesson_04_unit_testing/distance_unittest.py

```diff
@@ -0,0 +1,43 @@
+import unittest
```

Comment:
I know that it was not required to use pytest but try to write unit test using pytest. In modern python project you will most likely use pytest to write unit tests